documentation/modules/con-customizing-external-services.adoc

```diff
@@ -0,0 +1,51 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-customizing-deployments.adoc
+
+[id='con-customizing-external-services-{context}']
+= Customizing external Services
+
+When exposing Kafka to the outside of Kubernetes using load balancers or node prots, you can in addition to Labels and Annotations customize some more fields .
```

Comment:
typo "prots".
"you can customize some more fields in addition to Labels and Annotations" sounds better to me, but it's more for @PaulRMellor or @laidan6000